test cases/common/1 trivial/meson.build

```diff
@@ -1,8 +1,9 @@
 # Comment on the first line
 project('trivial test',
   # Comment inside a function call + array for language list
-  ['c'], default_options: ['buildtype=debug'],
-  meson_version : '>=0.52.0')
+  ['c'],
+  default_options: {'buildtype': 'debug', 'c_args': '-DFOO'},
```

Comment:
Would be nice to check in test.json that this prints the expected deprecation warning, and also `assert(get_option('c_args' == ['-DFOO']))`.